File: iOS_oud2/GC05_iOS/GC05_iOS/InschrijvingVorming.swift
import Foundation

class InschrijvingVorming
{
    var id: String
    var begindatum: Date
    var einddatum: Date
    var monitor: Monitor
    var vakantie: Vakantie
    
    init(id: String, begindatum: Date, einddatum: Date, monitor: Monitor, vakantie: Vakantie) {
        self.id = id
        self.begindatum = begindatum
        self.einddatum = einddatum
        self.monitor = monitor
        self.vakantie = vakantie
    }
}

File: Android/TI3_G05_JoetzApp2/app/src/main/java/com/hogent/ti3g05/ti3_g05_joetzapp/domein/InschrijvingVorming.java
package com.hogent.ti3g05.ti3_g05_joetzapp.domein;

import java.util.Date;

public class InschrijvingVorming {
    private int inschrijvingVormingID;
    private String monitor, vorming;
    private Date beginDatum, eindDatum;

    public InschrijvingVorming(){

    }

    public int getInschrijvingVormingID() {
        return inschrijvingVormingID;
    }

    public void setInschrijvingVormingID(int inschrijvingVormingID) {
        this.inschrijvingVormingID = inschrijvingVormingID;
    }

    public void setMonitor(String monitor){this.monitor = monitor;}
    public String getMonitor(){return monitor;}

    public void setVorming(String vorming){this.vorming = vorming;}
    public String getVorming(){return vorming;}

    public Date getBeginDatum() {
        return beginDatum;
    }

    public void setBeginDatum(Date beginDatum) {
        this.beginDatum = beginDatum;
    }

    public Date getEindDatum() {
        return eindDatum;
    }

    public void setEindDatum(Date eindDatum) {
        this.eindDatum = eindDatum;
    }
}
